gst/nnstreamer/include/tensor_filter_custom.h

```diff
@@ -99,6 +99,19 @@ typedef int (*NNS_custom_set_input_dimension) (void *private_data,
 typedef int (*NNS_custom_invoke) (void *private_data,
     const GstTensorFilterProperties * prop, const GstTensorMemory * input, GstTensorMemory * output);
 
+/**
+ * @brief Invoke the "main function" with flexible input and output. Output tensor memory should be allocated.
+ * @param[in/out] private_data A subplugin may save its internal private data here. The subplugin is responsible for alloc/free of this pointer.
+ * @param[in] info structure of input tensors info
+ * @param[out] info structure of output tensors info. The subplugin should fill this info.
+ * @param[in] input The array of input tensors. Allocated and filled by tensor_filter/main
+ * @param[out] output The array of output tensors. The subplugin should allocate the memory block for output tensor. (data in GstTensorMemory)
+ * @note rank limit (NNS_TENSOR_RANK_LIMIT) and typesize (tensor_element_size) defined in tensor_typedef.h
+ * @return 0 if success
+ */
+typedef int (*NNS_custom_invoke_dynamic) (void *private_data, const GstTensorsInfo * in_info, GstTensorsInfo * out_info,
+      const GstTensorMemory * input, GstTensorMemory * output);
```

Comment:
If this is used by custom_easy only, don't define it in custom. Move this definition to custom_easy.

Reply:
Moved to cusom_easy.h, Thanks.